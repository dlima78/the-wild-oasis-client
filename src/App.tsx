const App = () => (
  <div>
    <h1>The wild oasis</h1>
  </div>
)

export default App
