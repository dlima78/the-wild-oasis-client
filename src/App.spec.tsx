import { render, screen } from '@testing-library/react'
import App from './App'

describe('<App />', () => {
  test('should render the heading', () => {
    const { container } = render(<App />)
    expect(screen.getByRole('heading', { name: /the wild oasis/i }))
    expect(container.firstChild).toMatchSnapshot()
  })
})
