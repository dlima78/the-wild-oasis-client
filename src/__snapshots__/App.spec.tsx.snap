// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render the heading 1`] = `
<div>
  <h1>
    The wild oasis
  </h1>
</div>
`;
